app/controllers/v1/sessions_controller.rb

```diff
@@ -260,7 +260,12 @@ def login_stats(status, error = nil)
         Rails.logger.info("LOGIN_STATUS_SUCCESS, tags: #{tags}")
         StatsD.measure(STATSD_LOGIN_LATENCY, url_service.tracker.age, tags: tags)
       when :failure
-        tags_and_error_code = tags << "error:#{error.code}"
+        code = if error
```

Comment:
I can see this is done all over the place, so might not be something we should tackle with this PR, but it would be nice to refer to the error codes with descriptive values. I might recommend having these in `vets-api/lib/saml/responses/base.rb`, something like adding: `UNKNOWN_ERROR_CODE = '007'` at the top of that file, so in places like this we can refer to this string as: `SAML::Responses::Base::UNKNOWN_ERROR_CODE`